templates/networkInterface.json

```diff
@@ -8,7 +8,8 @@
     },
     "variables": {
         "subnetId": "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('networkInterfaceSetting').vnetName, parameters('networkInterfaceSetting').subnetName)]",
-        "appGatewayBackendPoolId": "[resourceId('Microsoft.Network/applicationGateways/backendAddressPools',parameters('networkInterfaceSetting').appGatewayName,parameters('networkInterfaceSetting').appGatewayBackendPoolName)]"
+        "lbBackendPoolIpv4Id": "[resourceId('Microsoft.Network/loadBalancers/backendAddressPools',parameters('networkInterfaceSetting').lbName,parameters('networkInterfaceSetting').lbBackendPoolIpv4Name)]",
```

Comment:
We have decided to use Application Gateway in develop branch. There is no load balancer anymore. I think we don't need to change this file.